cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -131,24 +131,27 @@ func NewDefaultComponentConfig(insecurePort int32) (componentconfig.CloudControl
 	return internal, nil
 }
 
-// AddFlags adds flags for a specific ExternalCMServer to the specified FlagSet
-func (o *CloudControllerManagerOptions) AddFlags(fs *pflag.FlagSet) {
-	o.CloudProvider.AddFlags(fs)
-	o.Debugging.AddFlags(fs)
-	o.GenericComponent.AddFlags(fs)
-	o.KubeCloudShared.AddFlags(fs)
-	o.ServiceController.AddFlags(fs)
+// Flags returns flags for a specific APIServer by section name
+func (o *CloudControllerManagerOptions) Flags() (fss apiserverflag.NamedFlagSets) {
+	o.CloudProvider.AddFlags(fss.FlagSet("cloudProvider"))
+	o.Debugging.AddFlags(fss.FlagSet("debugging"))
+	o.GenericComponent.AddFlags(fss.FlagSet("generic"))
+	o.KubeCloudShared.AddFlags(fss.FlagSet("kubeCloudShared"))
```

Comment:
Disregard my CamlCase comment. Why don't we use human readable section topics? There show up 1:1 in the output. So "Secure serving" instead of "secureServing". And "kubeCloudShared" should be added to "generic" maybe?

Reply:
i would like to keep split, but add `generic` also ok, for that two option have no boundary obviouly

Reply:
![image](https://user-images.githubusercontent.com/30410021/44266369-e788e700-a25c-11e8-9230-4dcbe06a95cf.png)
:rolling_on_the_floor_laughing:

Reply:
When output, thie first lettle would capital, seems ok
![image](https://user-images.githubusercontent.com/30410021/44266649-dd1b1d00-a25d-11e8-9717-6cfad468ee5d.png)